### ISSUE TITLE
[Feature] headless mode

### ISSUE THREAD

**Issue by Lukas1818** [rwthmoodle/Sync-my-L2P#8]
It would be a nice feature if Sync-my-L2P have a headless mode. This allow you to automatic download moodle, so you can use the learning material if moodel is down.

Current automatic backups are not very practical, because pop-ups appears and because you can not run Sync-my-L2P on a device without a desktop.

**Comment by t-schroeder** [comment]
I agree this would be a nice feature, but it would require rewriting much of the program to the point where it would probably be easier to just write a separate program or script for this. I don't think we will be able to do this any time soon.

**Comment by Lukas1818** [comment]
Because of this missing feature I had start using [syncMyMoodle](https://github.com/Romern/syncMyMoodle) instead. It can run headless.
Now a cronjob does automatic sync moodel every time if I start my pc. So the next several days failure of moodel can happen.

**Comment by mkroening** [comment]
Just for your information, I will shamelessly plug my own program for that use case here: [Edu Sync](https://github.com/edu-sync/edu-sync/blob/0259d3b264bbc87b1c11c47571d67a36780d85bc/README.md).

It does not support as many additional content types as syncMyMoodle (yet), but does support syncing basic contents from any Moodle instance (not just RWTH's) from the CLI.
